README.md

```diff
@@ -22,9 +22,11 @@ Each ARM firmware library is organized in the following structure:
     ├── cmsis
     │   └── gd
     │       └── gd32xxx
-    └── standard_peripheral
-        ├── include
-        └── source
+    ├── standard_peripheral
+    │   ├── include
+    │   └── source
+    └── support (optional)
+        └── GigaDevice.GD32xxx_DFP.1.0.0.pack (Only for packs not in [Keil MDK5 Software Packs](https://www.keil.com/dd2/pack/))
```

Comment:
Please remove binaries from this module (send a new PR dropping this). This is not acceptable, you can provide details on how to download suck packs (not to mention that GD is responsible to host them, and can be installed automatically via cmsis-pack-manager).

Reply:
IMO GD should be responsible of updating packages in CMSIS repo, so that the standard tool cmsis-pack-manager can be used. In fact, they seem to have most of them (A503 is new, maybe it's not there yet, time to ping vendor?).

Reply:
@gmarull ,

Why you are saying that the pack file is a binary file? As far I can see it is just a zip with source code and register definitions. You can just unzip <file> and read all the content.

Below is the full content of the file:

```
.
├── Device
│   ├── Firmware
│   │   └── Peripherals
│   │       ├── inc
│   │       │   ├── gd32a50x_adc.h
│   │       │   ├── gd32a50x_bkp.h
│   │       │   ├── gd32a50x_can.h
│   │       │   ├── gd32a50x_cmp.h
│   │       │   ├── gd32a50x_crc.h
│   │       │   ├── gd32a50x_dac.h
│   │       │   ├── gd32a50x_dbg.h
│   │       │   ├── gd32a50x_dma.h
│   │       │   ├── gd32a50x_exti.h
│   │       │   ├── gd32a50x_fmc.h
│   │       │   ├── gd32a50x_fwdgt.h
│   │       │   ├── gd32a50x_gpio.h
│   │       │   ├── gd32a50x_i2c.h
│   │       │   ├── gd32a50x_mfcom.h
│   │       │   ├── gd32a50x_misc.h
│   │       │   ├── gd32a50x_pmu.h
│   │       │   ├── gd32a50x_rcu.h
│   │       │   ├── gd32a50x_rtc.h
│   │       │   ├── gd32a50x_spi.h
│   │       │   ├── gd32a50x_syscfg.h
│   │       │   ├── gd32a50x_timer.h
│   │       │   ├── gd32a50x_trigsel.h
│   │       │   ├── gd32a50x_usart.h
│   │       │   └── gd32a50x_wwdgt.h
│   │       └── src
│   │           ├── gd32a50x_adc.c
│   │           ├── gd32a50x_bkp.c
│   │           ├── gd32a50x_can.c
│   │           ├── gd32a50x_cmp.c
│   │           ├── gd32a50x_crc.c
│   │           ├── gd32a50x_dac.c
│   │           ├── gd32a50x_dbg.c
│   │           ├── gd32a50x_dma.c
│   │           ├── gd32a50x_exti.c
│   │           ├── gd32a50x_fmc.c
│   │           ├── gd32a50x_fwdgt.c
│   │           ├── gd32a50x_gpio.c
│   │           ├── gd32a50x_i2c.c
│   │           ├── gd32a50x_mfcom.c
│   │           ├── gd32a50x_misc.c
│   │           ├── gd32a50x_pmu.c
│   │           ├── gd32a50x_rcu.c
│   │           ├── gd32a50x_rtc.c
│   │           ├── gd32a50x_spi.c
│   │           ├── gd32a50x_syscfg.c
│   │           ├── gd32a50x_timer.c
│   │           ├── gd32a50x_trigsel.c
│   │           ├── gd32a50x_usart.c
│   │           └── gd32a50x_wwdgt.c
│   ├── Include
│   │   ├── gd32a50x.h
│   │   └── system_gd32a50x.h
│   ├── Source
│   │   ├── ARM
│   │   │   └── startup_gd32a50x.s
│   │   └── system_gd32a50x.c
│   ├── Template
│   │   └── gd32a50x_libopt.h
│   └── Utilities
│       ├── gd32a503v_eval.c
│       └── gd32a503v_eval.h
├── Flash
│   ├── GD32A50x_128KB.FLM
│   ├── GD32A50x_256KB.FLM
│   └── GD32A50x_384KB.FLM
├── GigaDevice.GD32A50x_DFP.pdsc
└── SVD
    ├── GD32A503.SFR
    └── GD32A503.svd
```

I think the intention was keep easy to users until pack be available into cmsis-pack-manager and after that content will be dropped from here. The major concern was that pack is only available in the Chinese website version and it is not easy to find it if link is broke.

Reply:
The content is not binary, but a `zip` file is a binary from a git perspective. Binaries are well known to blow git history (see zephyr with board pictures), and remember Zephyr modules carry all their Git history.